src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -11,14 +11,15 @@ export default function FacilityAddress({
   clinicName,
   showPhone = true,
   level = '4',
+  isHomepageRefresh,
 }) {
   const address = facility?.address;
   const phone = facility?.telecom?.find(tele => tele.system === 'phone')?.value;
   const extraInfoClasses = classNames({
     'vads-u-margin-top--1p5': !!clinicName || !!phone,
   });
-  const Heading = `h${level}`;
-  const HeadingSub = `h${parseInt(level, 10) + 1}`;
+  const Heading = isHomepageRefresh ? `h2` : `h${level}`;
+  const HeadingSub = isHomepageRefresh ? `h2` : `h${parseInt(level, 10) + 1}`;
```

Comment:
I not sure about about HeadingSub so I made them all h2 when isHomePageRefresh is true. If the intent is for 'VA Appointment over the phone' to be h2 and all other headings to h3, I can change it. However, all the headings look the same to me.

![Screen Shot 2021-03-30 at 9 41 57 AM](https://user-images.githubusercontent.com/3587066/113009878-f981d480-913d-11eb-959f-30653b0a419a.png)

Reply:
I'm not sure you need the `isHomepageRefresh` prop? The `level` prop being passed in here should be different for the refresh vs not, and the phone headers should always be relative to what's in the facility address.